samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -67,7 +67,16 @@ private void Update()
         {
             mapComponent.APIKey = APIKey;
         }
+
         sceneLoadedCount = SceneManager.sceneCount;
+
+        if(currentSceneName == "ThirdPerson")
```

Comment:
So if the 3rd person sample is run individually, the user can't toggle cursor visibility. I feel like this sample specific logic should be in a component in the sample itself.

Reply:
I also thought about this but it's a little tricky to do the toggle in the third person script. Third person has [CursorLockMode.Locked](https://docs.unity3d.com/ScriptReference/CursorLockMode.Locked.html). When it's in that mode, the cursor is invisible regardless of the value of this property. Maybe it needs a new script just to toggle the cursor? I'm not sure.

Reply:
Ok so I had to disable the Cursor Locked option for this to work.

![image](https://github.com/Esri/arcgis-maps-sdk-unity-samples/assets/106696949/41c8abd4-e9c7-45cf-9947-9a1efab0e7da)

Reply:
now this should work as the same as Unreal.